ext_emconf.php

```diff
@@ -38,13 +38,7 @@
             'realurl' => '2.3.2-2.3.99',
             'themes' => '8.7.0-8.7.99',
             'gridelements' => '8.2.0-8.2.99',
-            'solr' => '7.5.1-7.5.99',
```

Comment:
Why you remove the dependencies from `ext_emconf.php`?
The dependencies from `composer.json` and `ext_emconf.php` should be the same.

Reply:
Sorry, is fixed.